### ISSUE TITLE
Invalid tar.gz created when there is a unicode filename

### ISSUE THREAD

**Issue by vti** [tokuhirom/Minilla#230]
When building a distribution with a unicode filename inside a .tar.gz is created but when unpacking: 

```
tar: Skipping to next header
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Removing the unicode file fixes the problem.

**Comment by skaji** [comment]
As mentioned in #228, I don't think minil should support such distributions.

OTOH, it could be nice if minil emits some helpful warnings then.

**Comment by vti** [comment]
Strange, I sometimes have unicode files for testing for example.

**Comment by karenetheridge** [comment]
Non-ascii characters in filenames in cpan distributions "should" be okay, as long as they are not files that get installed (lib/*.pm, bin/* etc), because we do not know what kind of filesystem the user will be installing on.  However, Test::Portability has a more limited character set that it checks for, to handle ideosyncracies on various filesystems.  Also see the Acme-LookOfDisapproval distribution for pitfalls arising from special characters in .pm files.

**Comment by skaji** [comment]
@karenetheridge Thanks for clarifying the current status of CPAN distributions which have non-ascii characters in their filenames.
It's very helpful.
